app/models.js

```diff
@@ -32,6 +32,25 @@ export type Grade = {
   name: string,
 };
 
+export type PhotoConsentDomain = 'WEBSITE' | 'SOCIAL_MEDIA';
+
+export type PhotoConsentSemester = 'SPRING' | 'AUTUMN';
+
+export type Semester = 'spring' | 'autumn';
```

Comment:
Do you really need both `PhotoConsentSemester` and `Semester`? Seems a bit overkill, don't you think?

Reply:
Semester is used multiple places in addition to Photo Consent. Changing Semester would require a lot og refactoring of out of scope features, to make it work everywhere.

Reply:
I wouldn't say "that" much refactoring, but anyways, I would then argue it's better to just use lowercase, because two separate (yet basically identical) types creates more confusing than anything else.

Reply:
I ended up using the Semester type, but refactoring it to uppercase.

Reply:
I ended up using lowercase Semester for photo_consents. To change the Semester type to UPPER_CASE would mean that the way we store semester for the bdb would need to be changed. To have Semster in UPPER_CASE in the bdb would make it inconsistent with the other types over there, thus only moving the problem.